virttest/qemu_monitor.py

```diff
@@ -1939,20 +1942,27 @@ def get_backingfile(self, device):
             pass
         return backing_file
 
-    def block_mirror(self, device, target, speed, sync, format, mode,
-                     cmd="drive-mirror", correct=True):
+    def block_mirror(self, device, target, sync, cmd="drive-mirror",
```

Comment:
@vivianQizhu 
Do you run any testcase for this PR.
I check this comment again.
Maybe i mislead you.
**hmp uses "drive_mirror",  "drive-mirror" is right for qmp.**